src/ontology/uberon-edit.obo

```diff
@@ -224361,6 +224375,18 @@ relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 relationship: in_taxon NCBITaxon:7898 ! Actinopterygii
 property_value: dcterms-date 2022-10-31T07:56:47Z xsd:dateTime
 
+[Term]
+id: UBERON:8480041
+name: testicular sheath
+def: "Bilayer sheath that covers both the insect male mature gonad and a portion of the genital disc. It is composed of an outer pigment cell layer, an inner muscle cell layer, and their respective basal laminae." [PMID:9553045]
```

Comment:
I am not sure it is exact that the sheath covers “a portion of the genital disc”. The Kozopas et al. (1998) reference says “covers both the mature gonad (the testis) and a portion of the genital disc (the seminal vesicle)”, but while the seminal vesicle develops from the genital disc, it’s not part of the genital disc. Here, the authors probably meant “a portion of genital disc-derived structures (the seminal vesicle)”.

Reply:
@gouttegd thanks for reviewing. I modified the definition according to your comment. If you think is best to remove this part of the sentence let me know